todo_app/data/test_app.py

```diff
@@ -0,0 +1,80 @@
+import os
+
+import pytest
+import requests
+from dotenv import find_dotenv, load_dotenv
+
+from todo_app import app
+
+
+@pytest.fixture
+def client():
+    # use our test integration .env.test config instead of the 'real' .env
+    file_path = find_dotenv(".env.test")
+    load_dotenv(file_path, override=True)
+
+    # create the new app
```

Comment:
As per your other comment - a hopefully simple solution is to ensure the mocking is in place here _before_ we create the app (and so before we initialise the TrelloService class). If we wanted more fine-tooth control over exactly what response was returned for each test we might have to add a bit more complexity (maybe we set some class level variables that individual tests can adjust) but for now we don't need to worry too hard about that
```python
def client(monkeypatch):
    ...

    # create the new app
    monkeypatch.setattr(requests, "get", stub)
    test_app = app.create_app()
```